Schutzfile

```diff
@@ -123,14 +123,14 @@
           {
             "title": "updates",
             "name": "updates",
-            "baseurl": "https://rpmrepo.osbuild.org/v2/mirror/public/f37/f37-x86_64-updates-released-20221124"
+            "baseurl": "https://rpmrepo.osbuild.org/v2/mirror/public/f37/f37-x86_64-updates-released-20230223"
```

Comment:
Let's revert the change in F37.  If all goes green, we can keep the changes for the other snapshots so we're testing newer versions until the F37 bug is fixed.